packages/2018-neighborhood-development/src/components/ExploreUrbanCampsiteSweeps/index.js

```diff
@@ -64,11 +64,8 @@ export class ExploreUrbanCampsiteSweeps extends React.Component {
             <div>
               <p>{contextualDesc}</p>
               <h2>{timer}</h2>
-              <BaseMap
-                initialLongitude={LONG}
-                initialLatitude={LAT}
+              <BaseMap initialLongitude={LONG} initialLatitude={LAT}>
                 initialZoom={ZOOM}
```

Comment:
`initialZoom` was pulled out of `BaseMap` somehow, so that should be added back in